config/serverless.oblt.yml

```diff
@@ -5,12 +5,9 @@ enterpriseSearch.enabled: false
 xpack.cloudSecurityPosture.enabled: false
 xpack.securitySolution.enabled: false
 
-## Enable the Serverless Obsersability plugin
+## Enable the Serverless Observability plugin
 xpack.serverless.observability.enabled: true
 
-# Onboarding
-xpack.observability_onboarding.ui.enabled: true
```

Comment:
This config was never shipped as part of a release? Asking if we can just remove it or if it could cause issues for existing cluster?

Reply:
@kuisathaverat will this be an issue?

Reply:
If the flag was not in 8.8 release and we have only internal test clusters using it, removing is acceptable.

Reply:
Thanks @sqren, that was my doubt, it was not released to customers but technically released in 8.8.

Reply:
FYI in the serverless Kibana deploy, we do not customize settings yet. So oblt clusters are not affected by this change.